mbofui/app/app.js

```diff
@@ -0,0 +1,224 @@
+var bofDataRef = new Firebase( 'https://flickering-fire-3313.firebaseio.com/bofs' );
+app.controller( 'mapController', [ '$scope', '$filter', '$timeout', '$log', 'leafletData', function ( $scope, $filter, $timeout, $log, leafletData ) {
+  angular.extend( $scope, {
+    center: {
+      zoom: 16,
+      autoDiscover: true
+    },
+    events: {},
+    controls: {
+      draw: {}
+    },
+    layers: {
+      baselayers: {
+        mapbox_light: {
+          name: 'Mapbox Light',
+          url: 'https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}',
+          type: 'xyz',
+          layerOptions: {
+            id: 'gsilver.pk8alhme',
```

Comment:
Also, is the token supposed to be private?  If so, it shouldn't be in this repo.  Move it to another file that doesn't get checked in and have this code refer to it.  (Yeah, I know it might be accessible in users' browsers anyway, but at least it's not as easy to access.)

And if `layers` doesn't change very much, how about moving them all out to another file, like `layer_config.js`?